Justfile

```diff
@@ -4,17 +4,18 @@ binary := "basic-validation-controller"
 default_namespace := "default"
 
 cluster-up:
-    kind create cluster --name {{cluster_name}} --image kindest/node:v1.19.1  --config ./kind-config.yaml
+    kind create cluster --name {{cluster_name}} --image kindest/node:v1.27.3  --config ./kind-config.yaml
```

Comment:
We can make kindest/node image version as variable, similar to other places in this MR!